framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -754,17 +755,48 @@ public AnnotationFormatter getAnnotationFormatter() {
      *
      * <p>
      * Subclasses cannot override this method; they should override
-     * {@link #createSupportedTypeQualifiers createSupportedTypeQualifiers} instead.
-
+     * {@link #createSupportedTypeQualifiers createSupportedTypeQualifiers}
+     * instead.
+     *
      * @see #createSupportedTypeQualifiers()
      *
-     * @return the type qualifiers supported this processor, or an empty
-     * set if none
+     * @return an immutable HashSet of the supported type qualifiers, or an
```

Comment:
Why the change to HashSet? It's a set and no ordering or concrete implementation is prescribed.